locations/spiders/giunti_al_punto_it.py

```diff
@@ -0,0 +1,28 @@
+from scrapy import Spider
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+
+
+class GiuntiAlPuntoITSpider(Spider):
+    name = "giunti_al_punto_it"
+    item_attributes = {"brand": "Giunti al Punto", "brand_wikidata": "Q76505309"}
+    start_urls = ["https://stores-giunti.retailtune.com/endpoint/sl/list.json.php"]
+    custom_settings = {
+        "DEFAULT_REQUEST_HEADERS": {"Authorization": "Basic Z2l1bnRpX2VuZHBvaW50X3VzZXI6eTBKeEZ5Z0ZwMFZkb0ExY3R2b2c="}
```

Comment:
Confirming I get the same key, hence it appears to be hard coded.

Reply:
Thanks